frontstage/views/surveys.py

```diff
@@ -241,6 +241,9 @@ def upload_survey(session):
         return render_template('surveys/surveys-upload-success.html', _theme='default', upload_filename=upload_filename)
     else:
```

Comment:
We probably want to log out something from the failed request. Maybe the status code at least?